changelog/0079-copy-invitation-supply-point-fields-to-location.yml

```diff
@@ -0,0 +1,27 @@
+title: Copy Supply Point data to Invitation's Location Field
+key: copy-invitation-supply-point-fields-to-location
+date: 2024-04-29
+optional_per_env: yes
+# (optional) Min version of HQ that MUST be deployed before this change can be rolled out (commit hash)
+min_commcare_version: c3f318f328372e904f4b1db42177309211ceb332
+# (optional) Max version of HQ that can be deployed before this change MUST be rolled out (commit hash)
+max_commcare_version: 72fc5f10dd31c42d854d20ed99d435ff3d2ad7a0
+context: |
+  The `supply_point` field in the `users_invitation` table is queried against the corresponding
```

Comment:
can you explain the _why_ more than the _what_ here? It would also be great to include how to determine whether this migration is even relevant (based on specific feature usage), or state explicitly that it is required regardless of feature usage.

Example change logs for a similar migration pattern:
https://commcare-cloud.readthedocs.io/en/latest/changelog/0063-backfill_sms_event_data_for_api_performance.html

Reply:
How's this [50c4952](https://github.com/dimagi/commcare-cloud/pull/6277/commits/50c49529df2752c9e10f3093f821ebc6e60ffa63)?